### ISSUE TITLE
Graphmap2 can't align dRNAseq reads with U

### ISSUE THREAD

**Issue by lpryszcz** [lbcb-sci/graphmap2#7]
Could you add U handling to graphmap? Then we won't have to convert U>T for dRNA-seq reads. 
Minimap2 handles it just fine ;)

**Comment by jmaricb** [comment]
@lpryszcz Yes, I will add the U support in the upcoming days. I will notify you when I do.
